<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestDataStream</class>
 <widget class="QMainWindow" name="TestDataStream">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1450</width>
    <height>850</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1450</width>
    <height>850</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1450</width>
    <height>850</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>楷体</family>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>TestDataStream</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>1441</width>
      <height>681</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab1">
     <attribute name="title">
      <string>文本</string>
     </attribute>
     <widget class="QPlainTextEdit" name="plainTextEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>1271</width>
        <height>621</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab2">
     <attribute name="title">
      <string>表格</string>
     </attribute>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>1421</width>
        <height>621</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>楷体</family>
        <pointsize>12</pointsize>
       </font>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1450</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar1">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actOpen"/>
   <addaction name="actSave"/>
   <addaction name="actOpenBin"/>
   <addaction name="actSaveBin"/>
   <addaction name="actTabReset"/>
   <addaction name="actAppend"/>
   <addaction name="actInsert"/>
   <addaction name="actDelete"/>
  </widget>
  <widget class="QToolBar" name="toolBar2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
   <addaction name="actOpenTxt"/>
   <addaction name="actSaveTxt"/>
   <addaction name="actModelData"/>
   <addaction name="actAlignLeft"/>
   <addaction name="actAlignCenter"/>
   <addaction name="actAlignRight"/>
   <addaction name="actFontBold"/>
   <addaction name="actExit"/>
  </widget>
  <action name="actOpen">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/open.bmp</normaloff>:/icons/open.bmp</iconset>
   </property>
   <property name="text">
    <string>打开stm文件</string>
   </property>
   <property name="toolTip">
    <string>打开预定义编码文件</string>
   </property>
  </action>
  <action name="actSave">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/save.bmp</normaloff>:/icons/save.bmp</iconset>
   </property>
   <property name="text">
    <string>保存stm文件</string>
   </property>
   <property name="toolTip">
    <string>保存为预定义编码文件</string>
   </property>
  </action>
  <action name="actAppend">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/append.bmp</normaloff>:/icons/append.bmp</iconset>
   </property>
   <property name="text">
    <string>添加行</string>
   </property>
   <property name="toolTip">
    <string>添加一行</string>
   </property>
  </action>
  <action name="actInsert">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/306.bmp</normaloff>:/icons/306.bmp</iconset>
   </property>
   <property name="text">
    <string>插入行</string>
   </property>
   <property name="toolTip">
    <string>插入一行</string>
   </property>
  </action>
  <action name="actDelete">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/delete.bmp</normaloff>:/icons/delete.bmp</iconset>
   </property>
   <property name="text">
    <string>删除行</string>
   </property>
   <property name="toolTip">
    <string>删除当前行</string>
   </property>
  </action>
  <action name="actExit">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/exit.bmp</normaloff>:/icons/exit.bmp</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
  </action>
  <action name="actModelData">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/import1.bmp</normaloff>:/icons/import1.bmp</iconset>
   </property>
   <property name="text">
    <string>文本显示</string>
   </property>
   <property name="toolTip">
    <string>模型数据显示到文本框里</string>
   </property>
  </action>
  <action name="actAlignLeft">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/508.bmp</normaloff>:/icons/508.bmp</iconset>
   </property>
   <property name="text">
    <string>居左</string>
   </property>
   <property name="toolTip">
    <string>文字左对齐</string>
   </property>
  </action>
  <action name="actAlignCenter">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/510.bmp</normaloff>:/icons/510.bmp</iconset>
   </property>
   <property name="text">
    <string>居中</string>
   </property>
   <property name="toolTip">
    <string>文字居中</string>
   </property>
  </action>
  <action name="actAlignRight">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/512.bmp</normaloff>:/icons/512.bmp</iconset>
   </property>
   <property name="text">
    <string>居右</string>
   </property>
   <property name="toolTip">
    <string>文字右对齐</string>
   </property>
  </action>
  <action name="actFontBold">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/500.bmp</normaloff>:/icons/500.bmp</iconset>
   </property>
   <property name="text">
    <string>粗体</string>
   </property>
   <property name="toolTip">
    <string>粗体字体</string>
   </property>
  </action>
  <action name="actTabReset">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/100.bmp</normaloff>:/icons/100.bmp</iconset>
   </property>
   <property name="text">
    <string>表格复位</string>
   </property>
   <property name="toolTip">
    <string>表格复位</string>
   </property>
  </action>
  <action name="actOpenBin">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/126.bmp</normaloff>:/icons/126.bmp</iconset>
   </property>
   <property name="text">
    <string>打开dat文件</string>
   </property>
   <property name="toolTip">
    <string>打开标准编码文件</string>
   </property>
  </action>
  <action name="actSaveBin">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/066.GIF</normaloff>:/icons/066.GIF</iconset>
   </property>
   <property name="text">
    <string>保存dat文件</string>
   </property>
   <property name="toolTip">
    <string>保存为标准编码文件</string>
   </property>
  </action>
  <action name="actOpenTxt">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/open.bmp</normaloff>:/icons/open.bmp</iconset>
   </property>
   <property name="text">
    <string>打开txt文件</string>
   </property>
   <property name="toolTip">
    <string>打开txt文件</string>
   </property>
  </action>
  <action name="actSaveTxt">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/save.bmp</normaloff>:/icons/save.bmp</iconset>
   </property>
   <property name="text">
    <string>保存txt文件</string>
   </property>
   <property name="toolTip">
    <string>保存txt文件</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actExit</sender>
   <signal>triggered()</signal>
   <receiver>TestDataStream</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>633</x>
     <y>453</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
